### PR TITLE
Yamato optimize

### PR DESCRIPTION
**Pull request by clusty**
Commits contain the list of changes.

### CODE REVIEW

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested package install on:
- 2020.2.1f1 (Windows only)

**Review by mfe — APPROVED** [review]
LGTM.

File: .yamato/upm-ci.yml [hunk 1]
```diff
@@ -89,9 +88,9 @@ build_centOS:
 pack:
   name: Pack
   agent:
-    type: Unity::VM::osx
-    image: package-ci/mac:latest
-    flavor: m1.mac
+    type: Unity::VM
```
Comment (judubu) [inline .yamato/upm-ci.yml:91]: I think I did the same on USD just before the holidays 👍

File: .yamato/upm-ci.yml [hunk 2]
```diff
@@ -186,24 +185,8 @@ testStandalone_{{ platform.name }}_{{ editor.version }}:
 
 test_trigger:
   name: TestsTrigger
```
Comment (judubu) [inline .yamato/upm-ci.yml:187]: Does that allow this job to run on any available platform?

**Review by judubu — APPROVED** [review]
That's a good idea. Thanks for the inspiration

File: .yamato/upm-ci.yml (hunk 2 above)
Comment (clusty) [reply .yamato/upm-ci.yml:187]: Ahh yes. by removing the agent, this job got tagged a a "virtual" job (basically a dependency gathering handle). There are no VMs associated with virtual jobs.

**Pull request merged**